pkg/apis/alicloud/types_infrastructure.go

```diff
@@ -110,7 +110,9 @@ type NatGatewayConfig struct {
 type InfrastructureStatus struct {
 	metav1.TypeMeta
 
-	VPC         VPCStatus
+	VPC VPCStatus
+
+	// Deprecated:  KeyPairName should not be used as ssh-key-pair will not generated by terraform.
```

Comment:
Hm, I see. Then let's add again similar rule to `.golangci.yaml`. This should do the trick.

```diff
diff --git a/.golangci.yaml b/.golangci.yaml
index d54e5358..8bb89792 100644
--- a/.golangci.yaml
+++ b/.golangci.yaml
@@ -5,3 +5,9 @@ run:
 linters:
   disable:
   - unused
+
+issues:
+  exclude-rules:
+  - linters:
+    - staticcheck
+    text: "SA1019:" # Excludes messages where deprecated variables are used
```

Reply:
I also constraint some files.